docs/extension_libraries.rst

```diff
@@ -14,5 +14,24 @@ library of Taichi, aiming at providing useful helper functions including:
 2. GLSL-alike vector functions like ``normalize``, ``distance``, ``reflect``.
 3. Well-behaved random generators including ``randUnit2D``, ``randNDRange``.
 4. Handy vector and matrix initializer: ``vec`` and ``mat``.
+5. Handy vector component shuffle accessor like ``v.xy``.
 
 Click here for `Taichi GLSL Documentation <https://taichi-glsl.readthedocs.io>`_.
+
+.. code-block:: bash
+
+    python3 -m pip install taichi_glsl
+
+
+Taichi THREE
+------------
+
+`Taichi THREE <https://github.com/taichi-dev/taichi_three>`_ or Tai3D is an
+extension library of Taichi to render 3D scenes into nice-looking 2D images
+in real-time (work in progress).
```

Comment:
```suggestion
`Taichi THREE <https://github.com/taichi-dev/taichi_three>`_ or `ti3` is an
extension library of Taichi to render 3D scenes in real-time (work in progress).
```

Cool! How about use `ti3` here so that people reach a consensus to use `import taichi_tree as ti3`.

Reply:
In fact I'm telling user to import using `t3`... Will we also tell them to `taichi_glsl` is `ts`?

Reply:
It's all your work, so you decide the abbreviation :-) 

My only suggestion is that, whichever abbreviation you pick, it's important to let your users know the standard one, just like everyone uses `import numpy as np` instead of as anything else.